weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CallsPage/CallsTable.tsx

```diff
@@ -140,6 +140,9 @@ export const CallsTable: FC<{
 
   paginationModel?: GridPaginationModel;
   setPaginationModel?: (newModel: GridPaginationModel) => void;
+
+  // Can include glob for prefix match, e.g. "inputs.*"
+  shownCols?: string[];
```

Comment:
Nit: I think this is operating more like a filter - if a column does not match this filter, is it not even selectable in the column picker. I might call this: `allowedColumnPatterns` or something a bit more specific.